Tasks/PublishBuildArtifactsV1/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 222,
-    "Patch": 1
+    "Patch": 2
```

Comment:
Probably, we should bump it up to 223 since today is Wednesday

Reply:
Oh, missed it, thanks!

Reply:
Fixed.